apps/backend/src/api/configuration_batch.py

```
@@ -0,0 +1,411 @@
+                detail=f"Batch deployment not found or cannot be cancelled: {batch_id}",
+            )
+
+        # Get updated status
+        status_result = await batch_service.get_batch_status(batch_id)
+
+        return BatchDeploymentCancelResponse(
+            batch_id=batch_id,
+            cancelled=True,
+            status=status_result.status if status_result else "unknown",
+            message="Batch deployment cancelled successfully",
+            rollback_initiated=True if status_result and status_result.rollback_plan else False,
```

Ruff (0.12.2)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)